apps/cli/src/commands/send/eip712.ts

```diff
@@ -0,0 +1,130 @@
+import { Hex, WalletClient } from "viem";
+import { Address } from "viem/accounts";
+
+const SERVICE_CONTEXT_PATH = "/transaction";
+
+type SendOptions = {
+    baseUrl: string;
+    chainId: number;
+    maxGasPrice: number;
+};
+
+const DEFAULT_SEND_CONFIG: Readonly<SendOptions> = {
+    baseUrl: "http://localhost:8080",
```

Comment:
Should this be http://localhost:8080/transaction? Isn't this duplicating the option config in `generic.ts`? (pardon my ignorance)